src/main/scala/editor/Editor.scala

```diff
@@ -1,18 +1,16 @@
 package editor
 
-object Editor {
-  val d = new SecretMaramaLogic("example");
+import maramafication.{MaramaficationLoader, MaramaficationManager}
 
+object Editor {
   def main(args: Array[String]): Unit = {
-    println(Hello() + " Marama!")
-  }
+    val loader = MaramaficationLoader("/src/main/scala/maramafication_data/test.mar")
+    val manager = MaramaficationManager.getInstance()
```

Comment:
Why would you assign the instance to a variable?
```
val manager = MaramaficationManager.getInstance()
```
Is'nt it better to just use the singleton instance where needed, and not use extra variables?

Reply:
Since the manager (mostly in the future) has to be used multiple times by the Editor I decided to assign it to a variable. This way it's easily (and clearly!) accessible. So for this particular piece of code it isn't needed, yet it's useful for future purpose.

Reply:
I don't agree with the reason that it is more clear, but if you think its better for future purposes than it is ok.